pints/toy/_stochastic_decay_model.py

```diff
@@ -0,0 +1,74 @@
+#
+# Logistic toy model.
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2019, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import numpy as np
+import pints
+
+from . import ToyModel
+
+
+class StochasticDecayModel(pints.ForwardModelS1, ToyModel):
+
+    """
+    Stochastic decay model of a single chemical reaction [1].
+
+    .. math::
+
+
+    Has one parameter: Rate constant :math:`k`.
+    The initial concentration :math:`A(0) = n_0` can be set using the

```

Comment:
Can you add the equations here to make this a bit more self-explanatory (i.e. equations that explain `k`, `A(t)` etc.).